kext/FileNVRAM/FileNVRAM.cpp

```diff
@@ -49,13 +51,26 @@ bool FileNVRAM::start(IOService *provider)
 {
     bool earlyInit = false;
     
-    LOG(NOTICE, "start() called (%d)\n", mInitComplete);
+    if (PE_parse_boot_argn("-NoFileNVRAM", gPEbuf, sizeof gPEbuf))
+    {
+        return false; /* following module disable method */
+    }
     
     //start is called upon wake for some reason.
-    if(mInitComplete)           return true;
+    if (mInitComplete) {
```

Comment:
For consistency with existing code, please use the following style:
The { should always be on a line by itself.
After an if statement, there should be no space between the if and the (:
if()
instead of
if ()